openstack/compute/api.go

```diff
@@ -27,6 +27,8 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/gorilla/mux"
+
+	"github.com/01org/ciao/payloads"
 )
 
```

Comment:
I'd like to avoid putting payloads into this file if possible. the Service interface implementation should be able to parse the openstack struct.